### PR TITLE
fix(assign):email information removal of assigned user

### LINKED ISSUE #530: Remove all email information about the assigned user on email templates
We don't want to provide more email information on the email we send to the user when he assigns a user, as well as other email notifications.

### Why
We have now a message system, so no need to share the e-mail in the email notification body

### Level
Beginner

### PR DESCRIPTION
**Pull request by jeevansrivastava**
## Description
This PR removes Email information for assigned user from following emails contents and both locale
Having subjects
1.Someone is interested in a task that you created on Gitpay
2. You have assigned {{name}} to do a task
## Changes
modules/mail/assign.js
locales/br.json
locales/en.json
locales/result/br.json
locales/result/en.json

## Issue

> Closes #530 

## Impacted Area

> AssignMail.owner.interested
> AssignMail.owner.assigned

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

-create an user
-Import an issue
-Create another user
-Apply to solve an issue in I'm interested (An email should be sent with subject 'Someone is interested in a task that you created on Gitpay')
-Log with the first user
-Assign user
-Accept from Second user (An email should be sent with subject 'You have assigned {{name}} to do a task')

### CODE REVIEW

File: locales/result/br.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "mail.payment.support": "User request support for Task",
   "mail.assign.owner.subject": "Alguém demonstrou interesse por sua tarefa no Gitpay",
   "mail.assign.owner.hello": "Olá",
-  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>. You can contact him on {{email}} to know more.",
+  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>.",
```
Comment (alexanmtz) [inline locales/result/br.json:20]: You shouldn't change the result files, they're managed by the translation library.

File: locales/result/en.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   "mail.payment.support": "User request support for Task",
   "mail.assign.owner.subject": "Someone is interested to solve an issue that you added on Gitpay.",
   "mail.assign.owner.hello": "Hello",
-  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>. You can contact him on {{email}} to know more.",
+  "mail.assign.owner.main": "{{name}} is interested to work on the task <a href=\"{{url}}\">{{title}}</a>.",
```
Comment (alexanmtz) [inline locales/result/en.json:20]: same here

**Comment by alexanmtz** [comment]
@jeevansrivastava actually your commit changes the original result files from the repository.

You need to update your changes in master, I recommend we close this PR and you start again with a fresh branch or only send the changes in https://github.com/worknenjoy/gitpay/blob/master/locales/en.json

**Comment by jeevansrivastava** [comment]
@alexanmtz you can close this PR will push changes with new branch

**Pull request closed without merging**